docs/markdown/glossary.md

```diff
@@ -0,0 +1,256 @@
+
+## Node
+
+A node is a participant in the Interledger network. It can be a sender, router, or receiver, which has an accounting relationship with one or more participants.                                                                
+
+## Packetized Payment
+
+A large payment which is split into smaller packets that are then sent sequentially. Packetized payments reduce the total amount of money in-flight at a given time, which can reduce certain risks and costs.                                                                      
+See also: [Streaming Payment](#streaming-payment)
+
+## Payment 
+
+In the context of ILPv4, a payment is understood to mean the transfer of value from the sender (payer) to the receiver (payee). Higher-level protocols may execute a "payment" by sending a series of ILP Packets whose sum is equal to the desired payment value.                                                                      
+
+## Payment Bandwidth
+
+See instead: [Bandwidth](#bandwidth) 
+
+## Payment Channel 
+
+A payment channel is a medium used by two parties to perform multiple value transactions amongst them, without sending these transactions to the underlying ledger. One or both parties will generally transfer some of their value assets on the ledger to the payment channel. Value transfers are then made between the two parties by exchanging "claims" that update their balance. When one of the parties wishes to close the channel, they submit the final balance to the ledger.
+                                      
+## Payment Pointer 
+
+A payment pointer is a standardized identifier for accounts that can receive payments. It is like an email address, but for sending/receiving value.
+
+A payment pointer resolves to an HTTPS URL that provides the location of a payment setup service endpoint at which a sender can initiate a payment to the receiver.
+
+For example, a pointer pointer such as `$example.com/bob` will resolve to `https://example.com/bob` endpoint URL.
+
+See also: [IL-RFC 26](https://github.com/interledger/rfcs/blob/master/0026-payment-pointers/0026-payment-pointers.md)                               
+
+## Peer    
+
+A router with which another router holds an account and exchanges routing information via the Interledger routing protocol.                                                                        
+
+## Peering 
+
+Routers "peer" (establish an account) with one another to exchange information used to determine the best route for a payment, and to route it. This is similar to how the internet service providers peer with each other (https://en.wikipedia.org/wiki/Peering).
+                                                                      
+## Prepare Phase   
+
+The first phase of an ILP payment, in which the ILP packet is passed from the sender, via the router(s), to the receiver. This represents an intent to pay, so long as the Fulfillment is delivered before the expiry.  
```

Comment:
> This represents an intent to pay

Prepayment is supported, too. My preference is to describe the Prepare as an IOU between two peers, which is enforced/accounted for in the balance if the Fulfill is returned, or is called off if the Reject is returned.